javatests/build/bazel/tests/integration/BUILD

```diff
@@ -6,3 +6,15 @@ bazel_java_integration_test(
     deps = ["@com_google_truth//jar"],
     data = ["//:root_bzl", "//tools:tools"]
 )
+
+java_library(
+    name = "BazelIntegrationTestNoSourcesSrc",
+    srcs = ["BazelIntegrationTestNoSourcesTest.java"],
+    deps = ["@org_junit//jar"] 
+)
+
+bazel_java_integration_test(
+    name = "BazelIntegrationTestNoSources",
+    runtime_deps = [":BazelIntegrationTestNoSourcesSrc"],
+    test_class = "build.bazel.tests.integration.BazelIntegrationTestNoSourcesTest",
```

Comment:
Name the test target BazelIntegationTestNoSourcesTest so we can remove the "test_class"

Reply:
This was actually on purpose since that's how we're going to use it on rules_scala.
Do you still want me to change it?

Reply:
wdyt?

Reply:
I don't think I understand the purpose. I am fine keeping it that way, but I'd like to have a comment with the reason in that case so we don't try to rename it.